articles/api/management/v2/tokens/programatically.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to programatically generate and use a token for the Management API
+section: apis
+crews: crew-2
+toc: true
+---
+# Programatically generate and use a token for the Management API
+
+In this article, we will show you how you can programmatically generate tokens for use with the Management API.
+
+The [manual process](/api/management/v2/tokens/manually) for getting a token might work for you if you want to test an endpoint or invoke it sporadically. However, if you need to make scheduled/frequent calls, then you should build a simple CLI tool that provides you with a token automatically. Its behavior is functionally equivalent to having a non-expiring token.
```

Comment:
This should be adjusted. You can get tokens for the Management API from all kind of apps, not only CLIs.